03-Design-Patterns-OOP/Movies.js

```diff
@@ -0,0 +1,210 @@
+function Movie(title, genre, minutes, rating) {
+    this.title = title;
+    this.genre = genre;
+    this.minutes = minutes;
+    this.rating = rating;
+    this.actor = [];
+
+    console.log("Se ha creado una instancia del objeto movie");
+};
+
+Movie.prototype.setTitle = function (title) {
```

Comment:
It might have be simpler to use the set(attr, value), get(attr) pattern you used for the Actor class, but nothing bad really, just another way of writing less code :)